### PR TITLE
Rename the repository to specify the name of the game

### PR DESCRIPTION
**Pull request by SoniaBarasa**
This pull request changes the name of the repository and specifies the name of the game. Previously, the repository's name suggested that it was for learning python programming.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Python_Programming_Learning
```
Comment (ahernank) [inline README.md:1]: You might also want to update the name here, so your README is fully up-to-date.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
-This repository is for general learning purposes. Aim is to learn conventional way to write a python code.
+This is a fun game that asks players to guess the capita cities of diffrerent countries in the world.
```
Comment (ahernank) [inline README.md:2]: Some minor spelling corrections here. You can update, `capita` -> `capital` & `diffrerent` -> `different`.

**Review by ahernank — COMMENTED** [review]
Thanks, @SoniaBarasa! Some small suggestions in your `README.md` file. 

For the name, I would change it to `country-capital-quiz-game`, (i.e. all lower case, with dashes) as sometimes it is confusing when certain words start with caps and others don't. Although there isn't a hard rule on using underscores vs dashes on names, I tend to stick to dashes for repository names (as they seem to be the most popular style and also useful on special repositories, for example, when dealing with websites).

**Comment by ahernank** [comment]
Another thought, here, you have updated the name of the repo but you will still have the old name on the local copy in your computer (i.e. local clone of the repo); as when we cloned it, we used an address that had the old name. 

Although not strictly necessary (as git can deal with this) it is a good idea to avoid confusion and update your local copy to have the new name.

To do this, first see the URLs git uses to communicate with your local copy using `git remote -v`. These ones will be using the old name. 

Then, you can use `git remote set-url origin NEW_URL` here, you want to replace NEW_URL with the new link git generates to  communicate with the repo (e.g. allowing things like cloning repositories). You can find this one on the same place we used when cloning this repo initially.

After you have updated this, you can re-run `git remote -v` to confirm the links reflect your new name.

File: README.md (hunk 2 above)
Comment (SoniaBarasa) [reply README.md:2]: @ahernank sure ,thank you for pointing out the typo

**Comment by SoniaBarasa** [comment]
Hello folks, am going to close this pull request because I have resolved the issue with the conflicting branches.

**Pull request merged**